test/CMakeLists.txt

```diff
@@ -62,5 +62,10 @@ add_lit_testsuite(test-dsa "Regression test for DSA"
 install (DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/simple DESTINATION share/seahorn/test)
 install (DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/solve DESTINATION share/seahorn/test)
 install (DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/abc DESTINATION share/seahorn/test)
-install (DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/dsa DESTINATION share/seahorn/test)
+install (DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/dsa DESTINATION share/seahorn/test
+  PATTERN "*/check_graphs.py" EXCLUDE
+  )
 install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/lit.cfg DESTINATION share/seahorn/test)
+install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/dsa/check_graphs.py DESTINATION share/seahorn/test/dsa
```

Comment:
use `install PROGRAMS` instead of `install FILES` to install binaries. See examples in [py/CMakeLists.txt](https://github.com/seahorn/seahorn/blob/master/py/CMakeLists.txt)